metricflow/test/snapshots/test_dataflow_to_sql_plan.py/SqlQueryPlan/Postgres/test_conversion_metric__plan0_optimized.sql

```diff
@@ -0,0 +1,87 @@
+-- Compute Metrics via Expressions
+SELECT
+  visit__referrer_id
+  , CAST(buys AS DOUBLE PRECISION) / CAST(NULLIF(visits, 0) AS DOUBLE PRECISION) AS visit_buy_conversion_rate
+FROM (
+  -- Combine Aggregated Outputs
+  SELECT
+    COALESCE(subq_18.visit__referrer_id, subq_28.visit__referrer_id) AS visit__referrer_id
+    , MAX(subq_18.visits) AS visits
+    , MAX(subq_28.buys) AS buys
+  FROM (
+    -- Aggregate Measures
+    SELECT
+      visit__referrer_id
+      , SUM(visits) AS visits
+    FROM (
+      -- Read Elements From Semantic Model 'visits_source'
+      -- Metric Time Dimension 'ds'
+      -- Pass Only Elements:
+      --   ['visits', 'visit__referrer_id']
+      SELECT
+        referrer_id AS visit__referrer_id
+        , 1 AS visits
+      FROM ***************************.fct_visits visits_source_src_10011
+    ) subq_17
+    GROUP BY
+      visit__referrer_id
+  ) subq_18
+  FULL OUTER JOIN (
+    -- Find conversions for EntitySpec(element_name='user', entity_links=()) within the range of count=7 granularity=TimeGranularity.DAY
```

Comment:
We can probably improve this message! I don't think the user needs to see any of the classes we have under the hood. Something like:
`Calculate conversions for '<qualified entity name>' within 7 days`
Or it could even include the names of the input measures like:
`Calculate conversions from 'visits' to 'buys' for 'user' within 7 days`